contrib/mesos/docs/ha.md

```diff
@@ -1,4 +1,24 @@
-## High Availability
+# High Availability
+
+Kubernetes on Mesos supports 2 HA modes:
+
+* Hot-standby
+* Cold-standby
+
+In the hot-standby mode, all master components (apiserver, controller manager, and scheduler) actively run 
```

Comment:
Right, that's also my understanding. It's not done yet, but the design is hot-standby. Maybe I should write a sentence with this caveat in this paragraph.